torch/optim/lr_scheduler.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, optimizer, last_epoch=-1):
                     raise KeyError("param 'initial_lr' is not specified "
                                    "in param_groups[{}] when resuming an optimizer".format(i))
         self.base_lrs = list(map(lambda group: group['initial_lr'], optimizer.param_groups))
+        self.last_epoch = last_epoch
```

Comment:
Is this even the correct fix? It gets quickly overwritten as `last_epoch + 1` in `step()`.

Reply:
@SsnL , this commit aims to revert to the old behavior where the attribute gets initiated in __init__ (versus step). Since step() could be over-rided regardless of overriding __init__, I think it's good practice to stick to initiating class attributes in __init__.

Reply:
Ah okay. It is kinda weird but I get it now. Thanks.